frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -761,15 +782,67 @@ export const CRDCardRow = connect(crdCardRowStateToProps)((props: CRDCardRowProp
   );
 });
 
+const InitializationResourceAlert: React.SFC<InitializationResourceAlertProps> = (props) => {
+  const { initializationResource, csv } = props;
+
+  const initializationResourceKind = initializationResource?.kind;
+  const { group: initializationResourceGroup } = groupVersionFor(
+    initializationResource?.apiVersion,
+  );
+  const model = modelFor(referenceFor(initializationResource));
+
+  // Check if the CR is already present - only checks for the specific name/ns in the initialization-resource
+  const [customResource, customResourceLoaded] = useK8sGet<K8sResourceCommon>(
+    model,
+    initializationResource?.metadata.name,
+    model.namespaced ? initializationResource?.metadata.namespace || csv.metadata?.namespace : '',
```

Comment:
The `?` should either go directly after `csv` or should be removed.

```suggestion
    model.namespaced ? initializationResource?.metadata.namespace || csv.metadata.namespace : '',
```

Reply:
You need to handle the case where `model` is null. This is a runtime error otherwise.